### PR TITLE
[10] Create & Style the Customer Signup Page using CSS

### PR DESCRIPTION
**Pull request by ainobs**
Screenshot (21)
Screenshot (22)
Capture
Capture1
Capture2

## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

**Comment by ainobs** [comment]
Capture3

**Comment by ainobs** [comment]
I've made the changes

File: submissions/victorAienobe/css/style.css [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        background-color: #ffffff;
+        margin:0;
+        padding:0;
+}
+.check1{
+        width:40%;
+        padding-top: 50px;
+}
+.pic {
+        padding-left: 2%;
```
Comment (adaugochi) [inline submissions/victorAienobe/css/style.css:12]: use `px` instead

File: submissions/victorAienobe/css/style.css [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+}
+.rectangle-1 {
+        width: 40%;
+        height: auto;
+        border-radius: 23px;
+        border: solid 1px #707070;
+        background-color: #ffffff;
+        margin-top:80px;
+        margin-bottom:20px; 
+        margin-left:15%;
```
Comment (adaugochi) [inline submissions/victorAienobe/css/style.css:65]: use `px` instead

File: submissions/victorAienobe/css/style.css [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+        background-color: #ffffff;
+        margin-top:80px;
+        margin-bottom:20px; 
+        margin-left:15%; 
+        padding:20px;       
+}
+.checkbox{
+        display:flex;
+        width:70%;
+        margin-left: 20%;
```
Comment (adaugochi) [inline submissions/victorAienobe/css/style.css:71]: same here

File: submissions/victorAienobe/css/style.css [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+        margin-top: 3%;
```
Comment (adaugochi) [inline submissions/victorAienobe/css/style.css:72]: same here

**Pull request merged**